### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by Sturmlilie**
Pending further reviews.

On merging, please use "Squash" option.

For translating "ping", I considered using "Signal" as is apparently canon in League of Legends: https://leagueoflegends.fandom.com/de/wiki/Signal
But decided against it as I'm not sure the word is commonly used today.

### CODE REVIEW

**Comment by FlashyReese** [comment]
Here are some changes from Ione 15 you can find him on Jellycord
```json
{
  "ping:key.categories.ping": "Ping",
  "key.ping": "Öffne Ping Wahlmenü",
  "ping.key.alert": "Ping (Warnung)",
  "ping.key.break": "Ping (Abbauen)",
  "ping.key.look": "Ping (Schauen)",
  "ping.key.goto": "Ping (Gehen)",
  "ping.key.attack": "Ping (Angreifen)",
  "ping.subtitle.bloop": "Ping",
  "ping.pingSelect.title": "Ping Auswahl",
  "ping.config.title": "Ping Konfiguration",
  "category.ping.general": "Allgemein",
  "ping.config.ping_accept_distance": "Maximale Ping Distanz",
  "ping.config.ping_duration": "Ping Dauer",
  "ping.config.ping_sound": "Ton",
  "category.ping.visual": "Darstellung",
  "ping.config.ping_red": "Rot",
  "ping.config.ping_green": "Grün",
  "ping.config.ping_blue": "Blau",
  "ping.config.block_overlay": "Block Umrahmung"
}
```

**Comment by Sturmlilie** [comment]
@FlashyReese Could you ask them to make a review on github? Otherwise it's really hard to follow their path of thought. Also in order to not have to manually diff.

**Comment by Sturmlilie** [comment]
For what it's worth, their translation violates the commonly accepted rules for forming compound nouns in German: https://www.duden.de/sprachwissen/sprachratgeber/Komposition

**Comment by FlashyReese** [comment]
I will see if I can get them to review on github, you have mentioned that their translations violate some common rules so I'm not sure if you still want their path of thought.

**Comment by Sturmlilie** [comment]
They have some good suggestions like "Umrahmung". It's just annoying to review when you don't know what they actually changed and what not.
